datastore/elastic_search/query.py

```diff
@@ -156,24 +183,35 @@ def delete_dictionary_records_by_word(connection, index_name, doc_type, entity_n
             '_op_type': 'delete',
         }
         str_query.append(delete_dict)
-        if len(str_query) > ELASTICSEARCH_BULK_HELPER_MESSAGE_SIZE:
+        if len(str_query) > constants.ELASTICSEARCH_BULK_HELPER_MESSAGE_SIZE:
             delete_bulk_queries.append(str_query)
             str_query = []
 
     if str_query:
         delete_bulk_queries.append(str_query)
 
     for delete_query in delete_bulk_queries:
-        helpers.bulk(connection, delete_query, stats_only=True, **kwargs)
+        result = helpers.bulk(connection, delete_query, stats_only=True, **kwargs)
+        ner_logger.debug('delete_entity_data_by_values: entity_name: {0} result {1}'.format(entity_name, str(result)))
 
 
-def add_data_elastic_search(
-    connection, index_name, doc_type, entity_name, word_variant_records, **kwargs
+def add_entity_data(
+    connection, index_name, doc_type, entity_name, value_variant_records, **kwargs
 ):
     """
+    Save entity data in ES for the records
+
+    Args:
+        connection: Elasticsearch client object
+        index_name: The name of the index
+        doc_type: The type of the documents that will be indexed
+        entity_name (str): name of the entity for which the data is to be created
+        value_variant_records (list): List of records to be created in ES
```

Comment:
Maybe mention where to refer for the format of elements in this list ?

Reply:
Added a sample dict